src/pages/Environments.js

```diff
@@ -426,16 +429,23 @@ const Environments = () => {
             headerRenderer: () => h(Sortable, { sort: diskSort, field: 'project', onSort: setDiskSort }, ['Billing project']),
             cellRenderer: ({ rowIndex }) => {
               const disk = filteredDisks[rowIndex]
-              const galaxyDiskNames = _.map(disk => disk.name, galaxyDisks)
-              const runtimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames), disks)
-              const runtimeDiskNames = _.map(disk => disk.name, runtimeDisks)
-              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, galaxyDiskNames) || disk.status === 'Deleting',
-                disksByProject[disk.googleProject]).length > 1
-              const multipleGalaxyDisks = _.remove(disk => _.includes(disk.name, runtimeDiskNames) || disk.status === 'DELETING',
+              const appDiskNames = _.map(disk => disk.name, appDisks)
+              const multipleRuntimeDisks = _.remove(disk => _.includes(disk.name, appDiskNames) || disk.status === 'Deleting',
                 disksByProject[disk.googleProject]).length > 1
+              const getAppType = diskName => {
+                // appDisks has appType populated on it if the disk was related to an app, but
```

Comment:
@salonishah11 this is another place where our life would be easier of "provisionedBy" was returned in the disk information. Then I would be able to tell if a disk was provisioned by an app without having to search in appDisks.